web_external/Viewer/index.jsx

```diff
@@ -79,13 +61,15 @@ class Viewer extends Component {
                                     ready: true
                                 });
                             }}
-                            key={this.state.itemModel.id} />,
+                            key={this.props.itemModel.id} />,
+                        <div className='no-annotation-message' key='no-annotation-message'>{this.props.itemModel && !this.props.annotationGeometryContainer && <span>No annotation</span>}
```

Comment:
Ack; please wrap this...

(In general, we need to discuss code style/formatting and avoid overly long lines, but this line has no reason why it should not be wrapped.)

Also, I think the entire `<div>` should be omitted if the messages is not shown?